.github/workflows/ci.yml

```diff
@@ -46,6 +46,7 @@ jobs:
         with:
           repository: "shopify/dawn"
           path: "./dawn"
+          ref: 'v6.0.0'
```

Comment:
I've introduced a fixed Dawn version, so we may ensure that an older version of Theme Check is running with a compatible Dawn version. Otherwise, Them Check fails with errors like `UnknownFilter`.